packages/api/wrangler.toml

```diff
@@ -14,6 +14,11 @@ no_bundle = true
 ENV = "dev"
 DEBUG = "true"
 DATABASE_URL = "http://localhost:3000"
+CARPARK_URL = "https://carpark-dev.web3.storage"
+
+[[r2_buckets]]
```

Comment:
we need prod and staging

Reply:
Added config for prod and staging buckets